src/plotting_static.jl

```diff
@@ -208,35 +205,40 @@ function plot_1D_solutions(res::Result; x::String, y::String, x_scale=1.0, y_sca
     sol_type, not_sol_type = _classify_plot_data(res, marker_classification)
 
     Ys, Yu = filter_solutions.(Y, res.classes[sol_type]), filter_solutions.(Y, [.!el for el in res.classes[sol_type]])
-    lines = ax.plot(X, Ys, marker) #Nan solutions are ignored 
+    lines = ax.plot(X, Ys, "o"; kwargs...) #Nan solutions are ignored 
     ax.set_prop_cycle(nothing) #reset color cycler state (NaN solutions aren't shown but the color cycler runs)
-    append!(lines,ax.plot(X, Yu, "X"))
+    append!(lines,ax.plot(X, Yu, "x"; kwargs...))
 
     
     if !isnothing(filename)
         xdata,ydata = [line.get_xdata() for line in lines], [line.get_ydata() for line in lines]
         markers = [line.get_marker() for line in lines]
-        marker_dict = Dict(marker=>sol_type,"X"=>not_sol_type)
+        marker_dict = Dict(marker=>sol_type,"x"=>not_sol_type)
         JLD2.save(_jld2_name(filename), Dict(string(x) => xdata,string(y)=>ydata,"marker_dict"=>marker_dict,"markers"=>markers))
     end
 
     ax.set_xlabel(latexify(x),fontsize=24) 
     ax.set_ylabel(latexify(_prettify_label(res,y)),fontsize=24) 
 
     #legend preparation
-    ignored_idx = [all(isnan.(line.get_ydata())) for line in lines] #make up a legend with only non ignored entries in the plotter
+    ignored_idx = [all(isnan.(line.get_ydata())) for line in lines] # make up a legend with only non ignored entries in the plotter
     Nb   = sum(.~ignored_idx) #number of branches
-    leg1 = ax.legend(string.(collect(1:Nb)),ncol=(Nb<=col_length) + (Nb÷col_length + 1)*(Nb>col_length),
```

Comment:
The removal of the col_length argument implies legends will appear longer than the panels. I suggest to put it back as an option (see for example the action of plot_1D_solutions on the two parametrons example)